js_task1/secondTask.js

```diff
@@ -0,0 +1,120 @@
+const generateButton = document.querySelector(".generate-btn");
+const checkButton = document.querySelector(".check-btn");
```

Comment:
Please, try to be consistent and change variables like generateBtn etc.

Reply:
done